lib/internal/fs/promises.js

```diff
@@ -129,6 +130,11 @@ function lazyFsStreams() {
   return fsStreams ??= require('internal/fs/streams');
 }
 
+function handleErrorFromBinding(error) {
+  ErrorCaptureStackTrace(error, handleErrorFromBinding);
```

Comment:
is there a perf difference when doing

```suggestion
  Error.stackTraceLimit && ErrorCaptureStackTrace(error, handleErrorFromBinding);
```

So only call ErrorCaptureStackTrace if Error.strackTraceLimit is not false (= 0)

Reply:
<details>
<summary>the code of benchmark</summary>

```js
'use strict';

const common = require('../common');
const fs = require('fs/promises');

const bench = common.createBenchmark(main, {
  n: [1e5],
  stacktraceLimit: [0, 10]
});


async function main({ n, stacktraceLimit }) {
  const fn = fs.access;
  Error.stackTraceLimit = stacktraceLimit;

  bench.start();
  for (let i = 0; i < n; i++) {
    try {
      await fn('no existance');
    } catch {}
  }
  bench.end(n);
}
```

</details>

The result of this benchmark on my machine is
```
// without Error.stackTraceLimit
fs/foo.js stacktraceLimit=0 n=100000: 25,569.003187542396
fs/foo.js stacktraceLimit=10 n=100000: 24,136.711560423188

// with Error.stackTraceLimit
fs/foo.js stacktraceLimit=0 n=100000: 25,789.21066371575
fs/foo.js stacktraceLimit=10 n=100000: 23,625.22690199484
```

Reply:
Thank you